Dockerfile

```diff
@@ -1069,8 +1168,10 @@ RUN \
 # Add Go programs to $PATH and sync timestamps to avoid rebuilds.
 RUN \
   ln -s ../libexec/go/bin/go /usr/bin/go && \
+  ln -s ../libexec/go-fips/bin/go /usr/bin/go-fips && \
```

Comment:
We can skip this symlink. Ideally the `/usr/bin/go` symlink would go away too since that's a potential footgun. However, most or all of our current spec files invoke Go by way of `go` and expect it to be in the path. The kubelet packages are an exception where the Go invocation is wrapped by a Makefile.

Please think on how you want to approach that at the build system level. One way might be to add `export PATH=/usr/libexec/go:${PATH}` to Twoliter's Dockerfile before the `rpmbuild` invocation, and then we can change that based on whether the FIPS feature flag is set for the variant.

Reply:
I was thinking I could wrap `go` invocation with a wrapper script that points to the appropriate toolchain depending on whether the FIPS feature flag is set.

I'll go ahead and remove the go-fips symlink. Once we have nailed down how to change behavior in twoliter, we can remove the original `/usr/bin/go` symlink.